dlpy/utils.py

```diff
@@ -1110,60 +1151,122 @@ def _convert_coco(size, box, resize):
     return (x_min, y_min, x_max, y_max)
 
 
-def _convert_xml_annotation(filename, coord_type, resize, name_file = None):
-    # always use en locale since we use this locale to generate our internal txt files
+def _convert_xml_annotation(filename, coord_type, resize, task = 'object detection', name_file = None):
+    # installing opencv2 is a painful experience in a variety of systems and cases,
+    # Ignore import cv2 when doing object detection task.
     try:
-        locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
```

Comment:
where is this locale now? it is not very easy to follow if you add chunks and remove chunks

Reply:
I put them in get_txt_annotation()